src/main/scala/org/apache/spark/sql/execution/datasources/oap/io/DataFile.scala

```diff
@@ -38,14 +38,15 @@ abstract class DataFile {
   def schema: StructType
   def configuration: Configuration
 
-  def getDataFileMeta(): DataFileMeta
-  def cache(groupId: Int, fiberId: Int): FiberCache
   def iterator(requiredIds: Array[Int], filters: Seq[Filter] = Nil)
-    : OapCompletionIterator[InternalRow]
+    : OapCompletionIterator[Any]
```

Comment:
These same changes already included in the previous pull request (/pull/3). Suggest to avoid.